rye/src/cli/rye.rs

```diff
@@ -52,12 +53,54 @@ pub struct Args {
     command: SubCommand,
 }
 
+#[derive(Clone, Debug, ValueEnum)]
+enum ShellCompletion {
```

Comment:
I just renamed this to match the terminology I saw in https://github.com/watchexec/watchexec/blob/a4df258735397bbb6465e1df0c42581a82a98049/crates/cli/src/args.rs#L1044C10-L1044C18.

Reply:
Sounds good!